
syntax = "proto3";

package mealprep.proto;


message ShoppingList {
  string meal_plan_id = 1;
  repeated ShoppingItem items = 2;
  int64 generated_at = 3;
}

message ShoppingItem {
  string ingredient_name = 1;
  float total_quantity = 2;
  string unit = 3;
  repeated string needed_for_recipes = 4;
}
